packages/react/src/wallet/ConnectWallet/screens/GetStartedScreen.tsx

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable @next/next/no-img-element */
```

Comment:
we should not have this rule in the `react` package in the first place, this is a `next` specific rule why is it enabled in our react package?

Reply:
yeah, this rule is added because we are using the `thirdweb` config. I can ignore the no-img-element rule

```js
// .eslintrc.cjs
module.exports = {
  root: true,
  extends: ["thirdweb"]
  rules: {
    "@next/next/no-img-element": "off", // <--
  },
};
```

Reply:
^ yeah we can start with that